docs/reference/troubleshooting.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[troubleshooting]]
+= Troubleshooting
+
+[partintro]
+--
+This section provides a series of troubleshooting guides aimed at helping the users
+fix various problems an Elasticsearch deployment might encounter.
+
+Reporting and diagnosing the problems presented in this section is normally assisted
+by the health API.
```

Comment:
```suggestion
Reporting and diagnosing the problems presented in this section is assisted
by the health API.
```

Perhaps link to the docs for the health API also?

Reply:
That didn't work for some reason I can't understand why :) I'll try to find out